### PR TITLE
Feature/cleanup

### PR DESCRIPTION
**Pull request by hez**

### CODE REVIEW

File: lib/shopify_api/rest/request.ex [hunk 1]
```diff
@@ -20,24 +20,18 @@ defmodule ShopifyAPI.REST.Request do
   end
 
   @spec get(AuthToken.t(), String.t()) :: {:error, any()} | {:ok, any()}
-  def get(auth, path) do
-    shopify_request(:get, url(auth, path), "", headers(auth), auth)
-  end
+  def get(auth, path), do: shopify_request(:get, url(auth, path), "", headers(auth), auth)
```
Comment (SteveAdams) [inline lib/shopify_api/rest/request.ex]: `put` and `get` are using slightly different formats. `get` is one line, `put` has `do:` on a new line. Is there a reason for that? (Maybe it means something different in Elixir?)

Comment (hez) [reply lib/shopify_api/rest/request.ex]: The code formatter enforces max line lengths, so sometimes the `do:` will end up on the next line.  Both are valid in Elixir.

these are all equivalent

```
def test do
   things
end

def test, do: things

def test,
  do: things
```

Comment (SteveAdams) [reply lib/shopify_api/rest/request.ex]: Nice, makes perfect sense. Thanks!

Comment (hez) [reply lib/shopify_api/rest/request.ex]: Thank you for looking at the PR! Welcome to the team!

**Pull request merged**